### PR TITLE
BREAKING CHANGE(web-twig): Remove HeaderDesktopActions color prop in favor of `isAtEnd` #DS-1059

### PR DESCRIPTION
**Pull request by crishpeen**

### CODE REVIEW

File: docs/migrations/web-react/MIGRATION-v2.md [hunk 1]
```diff
@@ -237,6 +238,31 @@ Examples:
   - `columnStart` = 1 + (12 - 6) / 2 = 4
   - `columnStart` = 1 + (12 - 4) / 2 = 5
 
+### Header: HeaderDesktopActions `isAtEnd` prop
+
+The `HeaderDesktopActions` component slots were simplified and the second slot alignment is the now
```
Comment (adamkudrna) [inline docs/migrations/web-react/MIGRATION-v2.md:243]:
```suggestion
The `HeaderDesktopActions` component slots were simplified and the second slot alignment is now
```

**Review by literat — APPROVED** [review]
LGTM 👍

File: packages/web-react/src/components/Header/useHeaderStyleProps.ts [hunk 2]
```diff
@@ -5,6 +5,7 @@ import { HEADER_COLOR_DEFAULT, HEADER_ACTIONS_COLOR_DEFAULT } from './constants'
 
 export interface UseHeaderStyleProps {
   color?: HeaderColorType;
+  isActionAtEnd?: boolean;
```
Comment (adamkudrna) [inline packages/web-react/src/components/Header/useHeaderStyleProps.ts:8]: I'm not sure if singular is better than (the incorrect) plural `Actions` mismatching the `is` verb…

What do you think?

```suggestion
  isActionsAtEnd?: boolean;
```

or

```suggestion
  hasActionsAtEnd?: boolean;
```

or

```suggestion
  isActionsSlotAtEnd?: boolean;
```

File: packages/web-react/src/types/header.ts [hunk 3]
```diff
@@ -34,7 +34,7 @@ export interface HeaderProps extends SpiritElementProps, ChildrenProps {
 export interface HeaderButtonProps extends SpiritButtonElementProps, ChildrenProps {}
 
 export interface HeaderDesktopActionsProps extends SpiritElementProps, ChildrenProps {
-  color?: HeaderActionsColorType;
```
Comment (adamkudrna) [inline packages/web-react/src/types/header.ts:37]: The type can be deleted I guess.

File: packages/web/src/scss/components/Header/_HeaderDesktopActions.scss [hunk 4]
```diff
@@ -13,7 +13,7 @@
     }
 }
 
-.HeaderDesktopActions--secondary {
+.HeaderDesktopActions--isAtEnd {
```
Comment (adamkudrna) [inline packages/web/src/scss/components/Header/_HeaderDesktopActions.scss:16]: How about just `.HeaderDesktopActions--end`? See `.Alert--center` as an example.

**Review by adamkudrna — APPROVED** [review]

File: packages/web-react/src/components/Header/HeaderDesktopActions.tsx [hunk 5]
```diff
@@ -3,12 +3,11 @@ import classNames from 'classnames';
 import { useStyleProps } from '../../hooks';
 import { HeaderDesktopActionsProps } from '../../types';
 import { useHeaderStyleProps } from './useHeaderStyleProps';
-import { HEADER_ACTIONS_COLOR_DEFAULT } from './constants';
 
 const HeaderDesktopActions = (props: HeaderDesktopActionsProps) => {
-  const { color = HEADER_ACTIONS_COLOR_DEFAULT, ...restProps } = props;
+  const { isAtEnd = false, ...restProps } = props;
```
Comment (pavelklibani) [inline packages/web-react/src/components/Header/HeaderDesktopActions.tsx:8]: in other react components we are defining default values above component like this:
```
const defaultProps = {
  isAtEnd: false,
};
```

and then in the component we do:
```
const propsWithDefaults = { ...defaultProps, ...props };
const { isAtEnd, ...restProps } = propsWithDefaults;
```

**Review by pavelklibani — APPROVED** [review]

File: packages/web-react/src/types/header.ts (hunk 3 above)
Comment (crishpeen) [reply packages/web-react/src/types/header.ts:37]: Nope still in HeaderDialogActionsProps

**Pull request merged**